### PR TITLE
Add Dockerfile and sentry.conf.py for production

### PR DESCRIPTION
**Pull request by joren485**

### CODE REVIEW

File: sentry.conf.py [hunk 1]
```diff
@@ -0,0 +1,309 @@
...
+                or 'postgres'
+            ),
+            'PASSWORD': (
+                env('SENTRY_DB_PASSWORD')
+                or env('POSTGRES_ENV_POSTGRES_PASSWORD')
+                or ''
+            ),
+            'HOST': postgres,
+            'PORT': (
+                env('SENTRY_POSTGRES_PORT')
```
Comment (thomwiggers) [inline sentry.conf.py]: why not use `env('SENTRY_POSTGRES_PORT', '')`?

**Review by thomwiggers — CHANGES_REQUESTED** [review]
The `env` environment variable loader can set a default value. No need to mess around with `or`.

File: sentry.conf.py (hunk 1 above)
Comment (joren485) [reply sentry.conf.py]: I used the default `sentry.conf.py` and added `THALIA_API_DOMAIN` and `THALIA_API_SECRET`. 

I changed the `env` calls (using one `or`) to use the default value parameter.

Comment (thomwiggers) [reply sentry.conf.py]: weird that they have it like that. Might be something for a PR on Sentry.

**Review by thomwiggers — APPROVED** [review]

**Comment by thomwiggers** [comment]
Will you be setting up Docker Hub?

**Comment by joren485** [comment]
Yes I will create an automatic build on docker hub

**Comment by thomwiggers** [comment]
Awesome :+1:

**Pull request merged**